src/platform/node/engines/dot.ts

```diff
@@ -235,6 +238,25 @@ digraph dependencies {
 			return d.promise;
 		}
 
+		private generateMD() {
+			logger.info('creating MD', this.paths.md);
+
+			let svgContent = fs.readFileSync(this.paths.svg).toString();
+			let d = q.defer();
+			fs.outputFile(
+				this.paths.md,
+				svgContent,
```

Comment:
you're getting the SVG content and printing it to a markdown file! This doesn't produce a valid md file.